example/captions.js

```diff
@@ -1,9 +1,10 @@
-const fs    = require('fs');
-const path  = require('path');
+/* eslint-disable no-console */
```

Comment:
disable this for the `example` directory https://stackoverflow.com/a/41316801/157629